sematic/ui/src/types/image.tsx

```diff
@@ -0,0 +1,39 @@
+import { Skeleton } from "@mui/material";
+import { useContext, useEffect, useState } from "react";
+import { CommonValueViewProps } from "./common";
+import { base64ArrayBuffer } from "src/base64ArrayBuffer";
+import { UserContext } from "src/appContext";
+
+export default function ImageValueView(props: CommonValueViewProps) {
+  const { valueSummary } = props;
+  const { bytes, mime_type } = valueSummary;
+  const { user } = useContext(UserContext);
+
+  const [ imageBase64, setImageBase64 ] = useState<string | undefined>(undefined);
+
+  useEffect(() => {
+    const headers: HeadersInit = new Headers();
+    if (user?.api_key) {
+      headers.set("X-API-KEY", user.api_key);
+    }
+    fetch(`/api/v1/storage/blobs/${bytes.blob}/data`, {headers: headers})
```

Comment:
Regarding your comment about using `useHttpClient`, it will have to be modularized to support non-JSON request before I can use it here.

Reply:
cc @chance-an